### PR TITLE
Cleaned up index.html

### PR DESCRIPTION
**Pull request by SalemC**
Removed explanatory comments, shifted elements around.

### CODE REVIEW

**Review by samharvey44 — APPROVED** [review]
Excellent cleanup.

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,22 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta charset="utf-8" />
+    
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
```
Comment (ted-marozzi) [inline public/index.html:13]: I think the comments are useful should we leave them in?

**Review by ted-marozzi — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (ted-marozzi) [reply public/index.html:13]: Given we have a lot of new starters

Comment (ted-marozzi) [reply public/index.html:13]: Make a decision on this then you are free to resolve the convo and merge whatever you decide

Comment (SalemC) [reply public/index.html:13]: Agreed, better for people to understand what's going on!

Fixed in 5e4bd7d

Comment (ted-marozzi) [reply public/index.html:13]: Out of interest, can you resolve the convo yourself or do i have too

Comment (SalemC) [reply public/index.html:13]: Looks like I can, just have! :)

**Review by ted-marozzi — APPROVED** [review]

**Comment by ted-marozzi** [comment]
Ah i have too re-review

**Comment by ted-marozzi** [comment]
Nice work

**Pull request merged**